app/pages/sign-in/sign-in.tsx

```diff
@@ -11,7 +11,8 @@ import { decryptWallet, selectKeysSlice } from '../../store/keys';
 export const SignIn: React.FC = () => {
   const history = useHistory();
   const dispatch = useDispatch();
-  const [password, setPassword] = useState<string | null>(null);
+  // const [password, setPassword] = useState<string | null>(null);
+  const [password, setPassword] = useState<string | null>('980aa096dd224bd69685583b363de2be');
```

Comment:
Just an easy placeholder for dev?

Reply:
Yep, this is one of the "to remove" items